packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -114,7 +114,21 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
                                         />
                                     )}
                                 </Text>
-                                <AdStatus is_active={!!is_advert_active} />
+                                {!payment_method_names ? (
+                                    <div className='p2p-my-ads__table-status-warning'>
+                                        <div style={{ marginRight: '0.8rem' }}>
+                                            <AdStatus is_active={!!is_advert_active} />
+                                        </div>
+
+                                        <Icon
+                                            icon='IcAlertWarning'
+                                            size={16}
+                                            className='cfd-dashboard__maintenance-icon'
+                                        />
+                                    </div>
+                                ) : (
+                                    <AdStatus is_active={!!is_advert_active} />
```

Comment:
What are we trying to achieve with this?

Reply:
Removed conditional render and todo comment added to inform Likhith what must be done, thx for notice!